.changelog/2268.txt

```diff
@@ -0,0 +1,9 @@
+```release-note:bug
+Deletes resource outside of Terraform for the following: 
+- resource/cloud_backup_schedule
+- resource/encryption_at_rest
+- resource/push_based_log_export
+- resource/search_deployment
+- resource/stream_connection
+- resource/stream_instance
+```
```

Comment:
that's not the format to use as it won't work fine when passed to the changelog file: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md

you need to have an entry per resource, e.g: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.changelog/2100.txt

in this case it would be:

```release-note:bug
resource/cloud_backup_schedule: Deletes resource outside of Terraform for the following
````

```release-note:bug
resource/encryption_at_rest: Deletes resource outside of Terraform for the following
````

etc.

Reply:
knit: this text sounds we're deleting the resource outside. maybe we could use something like:

Fixes logic when resource is deleted outside of Terraform

Reply:
@kanchana-mongodb could you have another look?

Reply:
@kanchana-mongodb could you please check the changelog one more time 😅 ?